contrib/agent/build.gradle

```diff
@@ -30,6 +30,9 @@ def agentBootstrapClasses = agentBootstrapPackageDir + '**'
 def agentRepackaged = "${agentPackage}.deps"
 
 dependencies {
+  compileOnly libraries.grpc_context
+  testCompile libraries.grpc_context
+  compile group: 'com.google.code.findbugs', name: 'annotations', version: '3.0.1u2'
```

Comment:
We use 'com.google.code.findbugs:jsr305:3.0.0' so probably we should use 3.0.0 here as well and be consistent. In order to keep consistency an variable for the findbugs_version in the main build file may help (similar with grpcContextVersion).

Also probably better to define the these in the libraries in the main build. Same probably for byte-buddy.